FindTown/FindTown/Presentation/MapScene/Map/MapViewController.swift

```diff
@@ -8,8 +8,42 @@
 import UIKit
 import FindTownCore
 import FindTownUI
+import NMapsMap
+import RxSwift
+import RxCocoa
 
 class MapViewController: BaseViewController {
+    
+    let mapView = NMFMapView()
+    let naviBarSubView = UIView()
+    let rightBarButton = UIBarButtonItem(image: UIImage(named: "favoriteBtn"),
+                                         style: .plain,
+                                         target: nil,
+                                         action: nil)
+    let addressButton = UIButton()
+    let mapToggle = MapSegmentControl(items: ["인프라", "테마"])
+    let detailCategoryView = MapDetailCategoryView()
```

Comment:
```suggestion
final class MapViewController: BaseViewController {
    
    private let mapView = NMFMapView()
    private let naviBarSubView = UIView()
    private let rightBarButton = UIBarButtonItem(image: UIImage(named: "favoriteBtn"),
                                         style: .plain,
                                         target: nil,
                                         action: nil)
    private let addressButton = UIButton()
    private let mapToggle = MapSegmentControl(items: ["인프라", "테마"])
    private let detailCategoryView = MapDetailCategoryView()
```  

final, private 붙일 수 있는 곳에는 붙이는게 좋아보입니다!!

Reply:
아하 넵 감사합니다 !